crates/rattler_conda_types/src/repo_data/mod.rs

```diff
@@ -132,8 +132,8 @@ pub struct PackageRecord {
 
     /// Package identifiers of packages that are equivalent to this package but from other
     /// ecosystems.
```

Comment:
Maybe add a comment explaining, the difference between `None` and `Some(Vec::new())`

Reply:
done!